tests/hooks/test_invoke_lsp_glob_guard.py

```
@@ -0,0 +1,427 @@
+
+
+# ---------------------------------------------------------------------------
+# Entry points: module-as-script and runpy
+# ---------------------------------------------------------------------------
+
+
+class TestModuleAsScript:
+    def test_runs_as_subprocess_allows_non_glob(self):
+        import subprocess
+
+        result = subprocess.run(
```

Ruff (0.15.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)